src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/commands.py

```diff
@@ -49,6 +55,14 @@ def load_command_table(self, _):
         g.command('show', 'get')
         g.command('delete', 'delete')
 
+    # account virtual network rule operations
+    with self.command_group('dls account virtual-network', dls_virtual_network_sdk, client_factory=cf_dls_account_virtual_network) as g:
+        g.custom_command('create', 'add_adls_virtual_network_rule')
+        g.command('update', 'update')
```

Comment:
This needs to be a generic update command. https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#generic-update-commands